### ISSUE TITLE
Unit.isFleeing()

### ISSUE THREAD

**Issue by ghost** [OreaDev/libBetterBot#65]

**Comment by TheCruxDev** [comment]
I totally agree to this! Either the bot should follow fleeing mobs itself or there should be an easy way to do this via the API. Checking for range isn't the best idea I think.

**Comment by clbrinky** [comment]
I think this wouldn't be very difficult. Does the bot have access to chat? every mob emotes when it runs away
